xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1557 @@
+        <value>
+          <see langword="true" /> if Windows Communication Foundation (WCF) is permitted to invoke interactive channel initializers; otherwise, <see langword="false" />.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.IClientChannel"/> interface.</remarks>
+
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.IClientChannel.BeginDisplayInitializationUI">
+      <MemberSignature Language="C#" Value="IAsyncResult IClientChannel.BeginDisplayInitializationUI (AsyncCallback callback, object state);" />
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance class System.IAsyncResult System.ServiceModel.IClientChannel.BeginDisplayInitializationUI(class System.AsyncCallback callback, object state) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#IClientChannel#BeginDisplayInitializationUI(System.AsyncCallback,System.Object)" />
+      <MemberSignature Language="VB.NET" Value="Function BeginDisplayInitializationUI (callback As AsyncCallback, state As Object) As IAsyncResult Implements IClientChannel.BeginDisplayInitializationUI" />
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.IClientChannel.BeginDisplayInitializationUI : AsyncCallback * obj -&gt; IAsyncResult&#xA;override this.System.ServiceModel.IClientChannel.BeginDisplayInitializationUI : AsyncCallback * obj -&gt; IAsyncResult" Usage="serviceChannelProxy.System.ServiceModel.IClientChannel.BeginDisplayInitializationUI (callback, state)" />
+      <MemberSignature Language="C++ CLI" Value=" virtual IAsyncResult ^ System.ServiceModel.IClientChannel.BeginDisplayInitializationUI(AsyncCallback ^ callback, System::Object ^ state) = System::ServiceModel::IClientChannel::BeginDisplayInitializationUI;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.IClientChannel.BeginDisplayInitializationUI(System.AsyncCallback,System.Object)</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.IAsyncResult</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="callback" Type="System.AsyncCallback" />
+        <Parameter Name="state" Type="System.Object" />
+      </Parameters>
+      <Docs>
+        <param name="callback">The method that is called when this method completes.</param>

```

Comment:
The typical language used elsewhere on this didn't seem to fit right so I refactored the sentence a little. Feel free to adjust if needed.
```suggestion
        <param name="callback">The <see cref="T:System.AsyncCallback" /> delegate that receives the notification of the completion of an asynchronous operation to display an initialization UI.</param>
```